examples/basic/confluent/control-centre.yaml

```diff
@@ -0,0 +1,27 @@
+---
+apiVersion: platform.confluent.io/v1beta1
+kind: ControlCenter
+metadata:
+  name: controlcenter
+spec:
+  tls:
+    secretRef: tls-group1
```

Comment:
Do we need this in the basic deployment? Not sure?

Reply:
nah, you're right... in light of our conv the other day, I've changed these to 'auto generated'... there is still a reference to the secret-tls in kustomize, but should be able to remove all of that once AC-58 is sorted.